docs/bluebear.md

```diff
@@ -0,0 +1,181 @@
+# nf-core/configs: BlueBEAR HPC Configuration
+
+Configuration for the BlueBEAR high performance computing (HPC) cluster at the University of Birmingham.
+
+## Setup
+
+To run a Nextflow pipeline, load the following modules,
+
+```bash
+module load bear-apps/2022b
+module load Nextflow/24.04.2
+```
+
+## Apptainer/Singularity
+
+BlueBEAR comes with Apptainer installed for running processes inside containers. Our configuration makes use of this
+when executing processes.
+
+We advise you create a directory in which to cache images using the following environment variable. For example,
```

Comment:
Maybe mention it's a good idea to put the `export` command in a `.bashrc` or similar?

Reply:
Done!